drivers/gpu/drm/i915/intel_initial_modeset.c

```diff
@@ -80,11 +80,8 @@ static bool use_connector(struct drm_connector *connector)
 			break;
 		}
 
-		connector->status = connector->funcs->detect(connector, true);
-		if (connector->status != connector_status_connected) {
```

Comment:
@yakuizhao  I have pushed a new commit,    but not very confirmed  about  the changes here.
I dont quite understand the code here,  if connector name appears  in cmdline, it means we assume monitor is connected,  why we need a detction here?    and connector->force = cl_mode->force seems unnecessary.

Reply:
@yakuizhao  Hi,  could you help to review  this patch, thanks.

Reply:
@lonzoc , The fb_get_options is one generic API to handle the options related with connectors. 
It has some other usages. In such case we can't assume that it is always connected.
So I think that we can refer to the function of drm_helper_probe_detect.

Reply:
Ok,  thanks for your guidance.